pyaerocom/plot/heatmaps.py

```diff
@@ -233,6 +234,11 @@ def df_to_heatmap(
         annotation information for rows
 
     """
+    warnings.warn(
+        "This function is no longer supported. It may be removed in future versions.",
+        DeprecationWarning,
+    )
+
```

Comment:
`_format_annot_heatmap` is a private/internal function, no warning needed.

Reply:
Ah, I see, the github preview is wrong. When I expand the hidden code, I see he correct function.